design/codebases.go

```diff
@@ -216,6 +216,9 @@ var _ = a.Resource("codebase", func() {
 		a.Params(func() {
 			a.Param("codebaseID", d.UUID, "Codebase Identifier")
 		})
+		a.OptionalPayload(func() {
+			a.Member("branch")
```

Comment:
This makes it:

```json
{
  "branch": "master"
}
```

?

Reply:
Yeah, I tested locally against prod-preview and this optional body is processed correctly

Reply:
To make it consistent... 

```json
{
    "data": {
        "attributes": {
           "branch": "master"
        }
    }
}
```

Reply:
ok, will fix